src/components/ProfileList/ModerationProfileListItem.tsx

```diff
@@ -47,15 +47,15 @@ const stateStatus = (state: string): React.ReactNode => {
   )
 }
 
-export const ModerationProfileListItem: React.FC<{ data: ProfileModel }> = ({
-  data,
+export const ModerationProfileListItem: React.FC<{ profile: ProfileModel }> = ({
+  profile,
```

Comment:
https://medium.com/raccoons-group/why-you-probably-shouldnt-use-react-fc-to-type-your-react-components-37ca1243dd13